app/experimenter/nimbus-ui/.eslintrc.js

```diff
@@ -0,0 +1,40 @@
+module.exports = {
+    extends: [
+        'plugin:@typescript-eslint/recommended',
+        'plugin:prettier/recommended',
+        'plugin:react/recommended',
+        'prettier',
+        'prettier/@typescript-eslint',
+        'prettier/react',
+    ],
+    plugins: ['react', '@typescript-eslint'],
+    env: {
+        browser: true,
+        es6: true,
+    },
+    parser: '@typescript-eslint/parser',
+    parserOptions: {
+        ecmaFeatures: {
+            jsx: true,
+        },
+        ecmaVersion: 2018,
+        sourceType: 'module',
+        project: './tsconfig.json',
+    },
+    rules: {
+        '@typescript-eslint/no-explicit-any': 'off',
+        '@typescript-eslint/explicit-function-return-type': 'off',
+        '@typescript-eslint/no-use-before-define': 'off',
+        '@typescript-eslint/no-non-null-assertion': 'off',
+        '@typescript-eslint/camelcase': 'off',
+        '@typescript-eslint/ban-ts-ignore': 'off',
+        '@typescript-eslint/no-empty-function': 'off',
+        'react/display-name': 'off',
+        'react/prop-types': 'off',
+    },
+    settings: {
+        react: {
+            version: "detect"
+        }
+    }
+};
```

Comment:
Will React also use this config during dev compilation and prod build?

Reply:
Oh also, does this pull in the  `app/.prettierrc`?

Reply:
> Oh also, does this pull in the `app/.prettierrc`?

Oh we can probably get rid of that that's probably very old.  Formatting configs for js should live in their relevant folders, this probably isn't being used anymore.

Reply:
😆 This is where I've been pointing my VS Code at. I would love it if we had a single ESLint and Prettier config instead of unique ones per-project.

Reply:
That would be nice but probably way overkill considering rapid will be deleted as soon as we hit parity and legacy is in maintenance mode.  I don't think you need to worry about anything other than `nimbus-ui`

Reply:
Oh no I was wrong, it does use .prettierrc for everything so we can just leave it there and treat it as a global formatting config, as you said 👍

Reply:
I don't think we can have a single eslint though since everything uses different versions of react and not all of them are typescript.

Reply:
> Will React also use this config during dev compilation and prod build?

Looks like everything is building and hosting correctly to me